src/server/app.js

```diff
@@ -0,0 +1,46 @@
+const { exec } = require('child_process');
+const express = require('express');
+const path = require('path');
+
+const PORT = 3000;
+const app = express();
+
+app.use(express.static('dist'));
+
+app.get('/login', (req, res) => {
+  res.sendFile(
+    path.join(__dirname + '/../../dist/login.html')
+  );
+})
+
+app.get('/signin', (req, res) => {
+  res.sendFile(
+    path.join(__dirname + '/../../dist/signin.html')
+  );
+})
+
+app.get('/', (req, res) => {
+  res.sendFile(
+    path.join(__dirname + '/../../dist/main.html')
+  );
+})
+
+app.get('/profile', (req, res) => {
+  res.sendFile(
+    path.join(__dirname + '/../../dist/profile.html')
+  );
+})
+
+app.get('/500', (req, res) => {
+  res.sendFile(
+    path.join(__dirname + '/../../dist/500.html')
+  );
+})
+
+app.get('/404', (req, res) => {
```

Comment:
Чтобы обработать запросы не существующих страниц
```suggestion
app.get('/*', (req, res) => {
```